src/foam/u2/detail/TabbedDetailView.js

```diff
@@ -63,8 +63,14 @@ foam.CLASS({
                 .start(self.Tabs)
                   .forEach(sections, function(s, i) {
                     if ( ! visibilities[i] ) return;
+                    var title$ = foam.Function.isInstance(s.title) ?
+                      foam.core.ExpressionSlot.create({
+                        obj$: self.data$,
```

Comment:
Does this still work if you do: obj: self.data ?

Reply:
Yes, pushed the change.